front/components/data_source/CreateOrUpdateConnectionSnowflakeModal.tsx

```diff
@@ -79,6 +80,7 @@ export function CreateOrUpdateConnectionSnowflakeModal({
       warehouse: "",
     });
     _onSuccess(ds);
+    onClose();
```

Comment:
I guess here you'll want to add `setIsOpen(false)`, no?

Reply:
just onClose does the trick 👍

Reply:
Oh, already called in `onClose`?